addons/grenades/stringtable.xml

```diff
@@ -376,26 +376,32 @@
         <Key ID="STR_ACE_Grenades_parachute_flare_description">
             <English>Type: Star Parachute Flare</English>
             <Polish>Typ: Gwiezdna Flara Spadochronowa</Polish>
+            <Italian>Tipo: Granata a paracadute con bengala</Italian>
         </Key>
         <Key ID="STR_ACE_Grenades_40mm_flare_white">
             <English>M583 Illumination Flare (White)</English>
             <Polish>M583 Flara oświetlająca (Biała)</Polish>
+            <Italian>M583 Granata con bengala (bianca)</Italian>
```

Comment:
Is it correct that `bianaca` is lowercase? The others are all first letter uppercase.